data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -155,24 +161,24 @@ def _aggregate_to_tracts(
         self, block_group_df: gpd.GeoDataFrame
     ) -> gpd.GeoDataFrame:
         # The tract identifier is the first 11 digits of the GEOID
-        block_group_df["tract"] = block_group_df.apply(
-            lambda row: row[self.GEOID_FIELD_NAME][0:11], axis=1
-        )
+        block_group_df["tract"] = block_group_df[self.GEOID_FIELD_NAME].str[:11]
```

Comment:
this is faster...

but more importantly, I'm not sure why we are doing this step. 

I believe `aggregate_to_tracts` is a holdover from when you were using block groups. here, we already have a dataframe at the census tract level, and so we very much can still `dissolve` it from a "can this be done?" but I think we don't have to

Reply:
@esfoobar-usds - any thoughts here?

Reply:
It's indeed a holdover from when we were census-block based and can be removed.